lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
looking closer, the way the watcher works assumes that the agent can be used as an even emitter (listened to) whether the agent has been started or not. but appmetrics can't be used that way. I have an idea to resolve this. @sjanuary can I rebase your branch and push a commit onto it?